### PR TITLE
Add Stretchlink.app v1.01

### PR DESCRIPTION
**Pull request by cdransf**
Add v1.01 of Brett Terpstra's StretchLink utility.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/stretchlink.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'stretchlink' do
+  version '1.01'
+  sha256 'ba19d498eb6100c1c9d7dee9da1a971b6025a78a66033488f92349d95ea82e0b'
+
+  url 'http://stretchlinkapp.com/download/StretchLink.zip'
+  name 'StretchLink'
+  homepage 'http://stretchlinkapp.com/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/stretchlink.rb]: `:commercial`.

**Comment by cdransf** [comment]
Updated the request per the example. Thanks!

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by cdransf** [comment]
Done

**Pull request merged**